packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -416,10 +394,10 @@ abstract contract MajorityVotingBase is
     /// @notice Internal function to execute a vote. It assumes the queried proposal exists.
     /// @param _proposalId The ID of the proposal.
     function _execute(uint256 _proposalId) internal virtual {
-        Proposal storage proposal_ = proposals[_proposalId];
-        proposal_.executed = true;
+        proposals[_proposalId].executed = true;
+
+        bytes[] memory execResults = dao.execute(_proposalId, proposals[_proposalId].actions);
```

Comment:
why don't you use here the `_executeProposal` function from ProposalBase

Reply:
Good catch! 
Fixed in fef75f11ee0c176faf7db29dbcad9a3e509e41e9